Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
     <PropertyGroup>
-        <Version>2.0.0.0</Version>
-        <AssemblyVersion>2.0.0.0</AssemblyVersion>
-        <FileVersion>2.0.0.0</FileVersion>
+        <Version>2.1.0.0</Version>
+        <AssemblyVersion>2.1.0.0</AssemblyVersion>
+        <FileVersion>2.1.0.0</FileVersion>
```

Comment:
Revert changes to this file, they are automatically updated when releasing a new build